include/deal.II/lac/lapack_templates.h

```diff
@@ -27,788 +27,744 @@
+          const char *                   jobvl,
+          const char *                   jobvr,
+          const char *                   sense,
           const dealii::types::blas_int *n,
-          const dealii::types::blas_int *nrhs,
-          const float *                  A,
-          const dealii::types::blas_int *lda,
-          const dealii::types::blas_int *ipiv,
-          float *                        b,
-          const dealii::types::blas_int *ldb,
-          dealii::types::blas_int *      info);
-  // Invert matrix from LU factorization
-  void
-  dgetri_(const dealii::types::blas_int *n,
-          double *                       A,
+          float *                        a,
           const dealii::types::blas_int *lda,
-          dealii::types::blas_int *      ipiv,
-          double *                       inv_work,
+          float *                        wr,
+          float *                        wi,
+          float *                        vl,
+          const dealii::types::blas_int *ldvl,
+          float *                        vr,
+          const dealii::types::blas_int *ldvr,
+          dealii::types::blas_int *      ilo,
+          dealii::types::blas_int *      ihi,
+          float *                        scale,
+          float *                        abnrm,
+          float *                        rconde,
+          float *                        rcondv,
+          float *                        work,
           const dealii::types::blas_int *lwork,
+          dealii::types::blas_int *      iwork,
           dealii::types::blas_int *      info);
+
   void
-  sgetri_(const dealii::types::blas_int *n,
-          float *                        A,
+  dgeevx_(const char *                   balanc,
+          const char *                   jobvl,
+          const char *                   jobvr,
+          const char *                   sense,
+          const dealii::types::blas_int *n,
+          double *                       a,
           const dealii::types::blas_int *lda,
-          dealii::types::blas_int *      ipiv,
-          float *                        inv_work,
+          double *                       wr,
+          double *                       wi,
+          double *                       vl,
+          const dealii::types::blas_int *ldvl,
+          double *                       vr,
+          const dealii::types::blas_int *ldvr,
+          dealii::types::blas_int *      ilo,
+          dealii::types::blas_int *      ihi,
+          double *                       scale,
+          double *                       abnrm,
+          double *                       rconde,
+          double *                       rcondv,
+          double *                       work,
           const dealii::types::blas_int *lwork,
+          dealii::types::blas_int *      iwork,
           dealii::types::blas_int *      info);
-  // Compute Cholesky factorization of SPD
```

Comment:
pity that we loose those comments here.

Reply:
This is the main downside. Unfortunately, the one thing LAPACK doesn't seem to offer in its documentation is a brief function description.

At the same time these descriptions are not enough to actually call the function (since one needs to know the size and meaning of the array arguments) so I think losing them is acceptable.

Reply:
yes, I fully agree.